### PR TITLE
Boring style fixes.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/ticker.py [hunk 1]
```diff
@@ -2410,26 +2410,26 @@ def tick_values(self, vmin, vmax):
             ticklocs = []
             if decade_min <= -1:
                 expo = np.arange(decade_min, min(0, decade_max + 1))
-                ticklocs.extend(list(10**expo))
+                ticklocs.extend(10**expo)
             if decade_min <= 0 <= decade_max:
                 ticklocs.append(0.5)
-            if decade_max >= 1:
+            if 1 <= decade_max:
```
Comment (jklymak) [inline lib/matplotlib/ticker.py]: huh?

File: lib/matplotlib/ticker.py [hunk 2]
```diff
@@ -2410,26 +2410,26 @@ def tick_values(self, vmin, vmax):
...
             ticklocs = []
             if decade_min <= -2:
                 expo = np.arange(decade_min, min(-1, decade_max))
                 newticks = np.outer(np.arange(2, 10), 10**expo).ravel()
-                ticklocs.extend(list(newticks))
+                ticklocs.extend(newticks)
             if decade_min <= 0 <= decade_max:
                 ticklocs.extend([0.2, 0.3, 0.4, 0.6, 0.7, 0.8])
-            if decade_max >= 2:
+            if 2 <= decade_max:
```
Comment (jklymak) [inline lib/matplotlib/ticker.py]: This is backwards to the way most people think.  I guess you want to write it this way so the numbers are in order, and I wouldn't hold up the PR on this, but...

Comment (anntzer) [reply lib/matplotlib/ticker.py]: Sorry, that was from a while ago, undoing both of these...

**Comment by jklymak** [comment]
Huh, appveyor has actually passed this PR, but it shows up as pending on GH. Merging as clearly passing.

**Pull request merged**